packages/algob/src/internal/deployer.ts

```diff
@@ -131,14 +129,10 @@ class DeployerBasicMode {
     const data = await readBinaryMultiSig(name);
     const program = new Uint8Array(Buffer.from(data as string, 'base64'));
     const logicSignature = decode(program) as RawLsig;
-
-    // dummy logic signature
-    const dummyProgram = new Uint8Array(56);
-    dummyProgram.fill(0);
-    const lsig = new logicsig.LogicSig(dummyProgram, []);
+    const lsig = DUMMY_LSIG; // dummy logic signature
```

Comment:
ah, you are modifying it in line 135. Sorry, I didn't notice it. This might be not secure, because you change the constant. 
Sorry, could you rollback?
```
function dummyLsig() {
    return algosdk.makeLogicSig(new Uint8Array(56), []);
}
```

Reply:
Okay will rollback. Sorry, i should have noticed it too :/